### ISSUE TITLE
update create_feed for DOM changes

### ISSUE THREAD

**Issue by boblannon** [sunlightlabs/fara#43]
DOM must have changed.  `parse_and_save` is choking:

```
AttributeError: 'NoneType' object has no attribute 'find_all'
```

see https://github.com/sunlightlabs/fara/blob/master/fara_feed/management/commands/create_feed.py#L220

**Comment by boblannon** [comment]
actually, it looks like what's needed is cookie management

**Comment by boblannon** [comment]
fixed by 7af22e7b
